src/app/AppLayout/AppLayout.tsx

```diff
@@ -157,9 +160,20 @@ const AppLayout: React.FunctionComponent<IAppLayout> = ({children}) => {
     return () => sub.unsubscribe();
   }, [serviceContext.target]);
 
-  const handleLogout = React.useCallback(() =>
-    serviceContext.login.setLoggedOut(),
-    [serviceContext.login]);
+  const handleLogout = React.useCallback(() => {
+      const sub = serviceContext.login.setLoggedOut(token, authMethod).subscribe();
```

Comment:
The `Login.service` already knows the `token` and `authMethod`, doesn't it? So the `setLoggedOut()` call could/should just use those already-known details, and the `AppLayout` component wouldn't need to hold these new pieces of state.